docs/admin/maintenance/noble_migration.rst

```diff
@@ -68,6 +68,23 @@ The process will upgrade your application server first and then the monitor serv
 Once it finishes, you should verify you can submit tips via the Source Interface and can log into the
 Journalist Interface and download submissions.
 
+It is important that running the migration script, ``./securedrop-admin noble_migration``, completes
+successfully. If it fails, it is safe to run multiple times. If it continues failing, see :ref:`the debugging advice below<debugging>`.
+
+Verifying success
+^^^^^^^^^^^^^^^^^
+
+You can verify that the upgrade succesfully finished on the *Application Server* by running:
```

Comment:
```suggestion
You can verify that the upgrade completed successfully on the *Application Server* by running:
```

I don't think the original was wrong, but it didn't feel quite right to me. Please feel free to take or leave this suggestion.